src/api/json/catalog.json

```
@@ -1873,6 +1873,16 @@
       "fileMatch": ["function.json"],
       "url": "https://json.schemastore.org/function.json"
     },
+    {
+      "name": "Fusion resource pack metadata",
+      "description": "Fusion properties for Minecraft resource pack metadata",
+      "url": "https://json.schemastore.org/fusion-pack-metadata.json"
```

I think it would be good to be consistent and keep the `fileMatch` property, even if it is empty. We have had some past issues with trailing commas breaking some editors, and I just want to be sure

```suggestion
      "fileMatch": [], "url": "https://json.schemastore.org/fusion-pack-metadata.json"
```